src/components/PlayClassCard/PlayClassCard.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import "../PlayClassCard/PlayClassCard.css";
+import { IoPlay, IoStop, IoPause, IoHeart } from "react-icons/io5";
+import { useState } from "react";
+
+export default function PlayClassCard(props) {
+  const [isFavourite, setIsFavourite] = useState("false");
+  const [isActive, setIsActive] = useState(0);
+
+  const {
+    name,
+    isToggled,
+    isPlaying,
+    isStopped,
+    isPaused,
+    prepTime,
+    duration,
+    ...rest
+  } = props;
+
+  const toggleHeartIcon = () => {
+    setIsFavourite(!isFavourite);
+  };
+
+  function onPauseClick() {
+    setIsActive(1);
+  }
+
+  function onPlayClick() {
+    setIsActive(2);
+  }
+
+  function onStopClick() {
+    setIsActive(3);
+  }
+
```

Comment:
Very good.

Reply:
Took me some time to figure it out though 🙃